log_drains.go

```diff
@@ -23,3 +34,32 @@ func (c *Client) LogDrainsList(app string) ([]LogDrain, error) {
 	}
 	return logDrainsRes, nil
 }
+
+type LogDrainAddParams struct {
```

Comment:
The MVP we designed states that the payload for the API is:

```json
{
  "log_drain": {
    "type": "datadog",
    "token": "token",
    "region": "region",
  }
}```

It does not feel like you respect it